newIDE/app/src/ObjectsRendering/PixiResourcesLoader.js

```diff
@@ -165,12 +165,13 @@ export default class PixiResourcesLoader {
   ) {
     const loadedTexture = loadedTextures[resourceName];
     if (loadedTexture && loadedTexture.textureCacheIds) {
+      // The property textureCacheIds indicates that the PIXI.Texture object has some
+      // items cached in PIXI caches (PIXI.utils.BaseTextureCache and PIXI.utils.TextureCache).
+      // PIXI.Assets.unload will handle the clearing of those caches.
       await PIXI.Assets.unload(loadedTexture.textureCacheIds);
-      // Needed for video objects. Optional for other object types.
-      delete loadedTextures[resourceName];
```

Comment:
so no delete anymore?

Reply:
Nope, it was a bit forceful to do this. But it can be prevented